frontend/src/contest/submission/ResultView.tsx

```diff
@@ -1,17 +1,55 @@
 import React from "react";
 import { Trans } from "@lingui/macro";
-import { Alert } from "src/types/contest";
+import { Alert, Subtask } from "src/types/contest";
 
 type Props<T> = {
   cases: T[];
   alerts: Alert[];
+  subtasks?: Subtask[];
   renderCase: (c: T, i: number) => React.ReactNode;
   renderCaseSummary: (c: T, i: number) => React.ReactNode;
 };
 
 export function ResultView<T>({
-  cases, alerts, renderCase, renderCaseSummary,
+  cases, alerts, subtasks, renderCase, renderCaseSummary,
 }: Props<T>) {
+  let summary = subtasks ? (
```

Comment:
ideally, split the ternary into two parts (i.e. by making functions), and use `const`

Reply:
Fixed